dlk/python/dlk/templates/src/func/mean.cpp

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2019 The Blueoil Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "global.h"
+#include "func/mean.h"
+#include "time_measurement.h"
+
+void func_Mean(const TensorView<T_FLOAT, MemoryLayout::NHWC>& input,
+    const TensorView<int, MemoryLayout::C>& factor,
```

Comment:
What the name `factor` means?

Reply:
It is actually axis, I will change it in next commit